lib/Doctrine/Inflector/InflectorFactory.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Inflector;
+
+use Doctrine\Inflector\Rules\English;
+use Doctrine\Inflector\Rules\French;
+use Doctrine\Inflector\Rules\NorwegianBokmal;
+use Doctrine\Inflector\Rules\Portuguese;
+use Doctrine\Inflector\Rules\Spanish;
+use Doctrine\Inflector\Rules\Turkish;
+use InvalidArgumentException;
+use function sprintf;
+
+final class InflectorFactory
+{
+    public static function create(string $language = Language::ENGLISH) : Inflector
```

Comment:
How about making these __invoke()?

Reply:
It would look like this then?

```php
$inflector = (new InflectorFactory())(Language::ENGLISH);
```

Reply:
yep, for example

Reply:
For the end user, do you think that is a better experience compared to:

```php
$inflector = InflectorFactory::create(Language::ENGLISH);
```

It seems not as good but maybe it is just because I am not used to it.

Reply:
Pushed up the change.